docs/tutorial.rst

```diff
@@ -934,11 +959,11 @@ Next is "recapitation",
 for which the rationale and method is described in detail in the
 `pyslim documentation <https://tskit.dev/pyslim/docs/latest/tutorial.html#recapitation>`__.
 The third (and least crucial) step is to *simplify* the tree sequence.
-If as above we ask for 200 samples from a population whose final size is
+If as above we ask for 200 haploids from a population whose final size is
```

Comment:
```suggestion
If as above we ask for 100 sampled individuals from a population whose final size is
```
The command above has `100`, so saying "200 haploids" is confusing for no reason, I think?